orca/src/runner/search.rs

```diff
@@ -30,7 +30,7 @@ struct CsvRow {
 
 fn basic_arbiter(
     mut broadcast_rx: tokio::sync::broadcast::Receiver<TestPhase>,
-    raw_results_rx: crossbeam::channel::Receiver<(Duration, Duration, usize)>,
+    raw_results_rx: &crossbeam::channel::Receiver<(Duration, Duration, usize)>,
```

Comment:
What is the difference between `&crossbeam::channel::Receiver<(Duration, Duration, usize)>` and `crossbeam::channel::Receiver<(Duration, Duration, usize)>`?

Reply:
Again clippy - rather than pass the whole thing, pass by reference to save stack/call space.